packages/utils/src/conversions.ts

```diff
@@ -220,3 +220,49 @@ export function bufferToHexadecimal(value: Buffer | Uint8Array): string {
 
     return Buffer.from(value).toString("hex")
 }
+
+/**
+ * Converts a buffer to a base64 string. It accepts 'Buffer' or 'Uint8Array'.
+ * @param value The buffer to convert.
+ * @returns The converted base64 string.
+ */
+export function bufferToBase64(value: Buffer | Uint8Array): string {
```

Comment:
"bytes" instead of "buffer" maybe?  Since the use of the Buffer type is optional for this function and its inverse.

Reply:
If you're going to refer to bytes not buffer shouldn't that be reflected in the function name, not just the comments?  I.e. consider renaming the functions to byteToBase64 and base64ToBytes?

Reply:
Well, we should update many names in this library in that case. I actually think we can consider Uint8Array to be a type of buffer in this context no?

Reply:
At the code level I think it's the opposite?  Buffer is a type of Uint8Array.

If you mean "buffer" as an english word, then yes, a Uint8Array is a way of representing a "buffer" of bytes, but I'd say that's a confusing definition to rely on when there's a class called Buffer involved.

This feels like bikeshedding, though, and I don't really care enough for it to be worth arguing.

Reply:
I think it makes sense, as "Buffer" in the function name may suggest it's a "Buffer" type. I'd keep this small inaccuracy tho since we would need to change many names to keep it consistent and it would be a breaking change.